code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -769,16 +769,16 @@ var/global/list/obj/item/device/pda/PDAs = list()
 	else
 		L = get(src, /mob/living/silicon)
 
-	if(L)
+	if(L && L.stat != UNCONSCIOUS)

```

Comment:
Well I mean, ghosts can see all PDA messages anyway, so it doesn't really matter.